lib/src/authorization_code_grant.dart

```diff
@@ -59,6 +59,11 @@ class AuthorizationCodeGrant {
   /// documentation.
   final Uri authorizationEndpoint;
 
+  /// The scope strings will be separated by the provided [delimiter]. This
+  /// defaults to `" "`, the OAuth2 standard, but some APIs (such as Facebook's)
+  /// use non-standard delimiters.
+  final String delimiter;
```

Comment:
This should be private, and the constructor parameter should be documented independently.